python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -102,7 +105,7 @@ def fetchReferences(self, dataRef, exposure):
     def _makeArgumentParser(cls):
         parser = lsst.pipe.base.ArgumentParser(name=cls._DefaultName)
         parser.add_id_argument("--id", "deepCoadd_forced_src", help="data ID, with raw CCD keys + tract",
-                               ContainerClass=lsst.coadd.utils.CoaddDataIdContainer)
+                               ContainerClass=lsst.coadd.utils.ExistingCoaddDataIdContainer)
```

Comment:
I understand at least part of why @PaulPrice started using this (it makes it easier to deal with patches where there are no data), but I'm not totally sold on whether silently skipping data IDs where coaddition failed is the right logic; it seems like that could lead to problems being discovered later than they ought to be.

@laurenam, do you know if this change is necessary (rather than just a convenience) for the rest of this issue?  If not, I'd be inclined to suggest we take it off this issue and RFC it before we bring it over.  If so, I think we bring it over now but open an issue for reconsidering it in the future.